### DISCUSSION TITLE
change the software localization

### DISCUSSION THREAD

**Discussion by edmaSB** [orchidsoftware/platform#2749] [Q&A]
I'm having trouble changing the default location of the software. I changed the Laravel configuration but some sentences remain in English.
What are the steps to change the location?

**Comment by c-gross** [comment]
Only thing to change is in the config/app.php locale I think.
If some elements are not translated I would assume that a few translations are just missing.

You can add them in our lang folder. 
Or you add them to the orchid translations ([orchid/platform/resources/lang](https://github.com/orchidsoftware/platform/tree/master/resources/lang)) and make a pull request to add them to the code.
